SIL.Core/Threading/GlobalMutex.cs

```diff
@@ -28,10 +28,12 @@ public class GlobalMutex : DisposableBase
 		public GlobalMutex(string name)
```

Comment:
Do we need to document that (on Mac) the actual mutex might not be named as requested? (Probably not important since it will always work like this.)